rust/src/lib/nispor/mod.rs

```diff
@@ -1,4 +1,7 @@
+// SPDX-License-Identifier: Apache-2.0
+
 mod apply;
+mod dns;
```

Comment:
`cargo-fmt  --check` failed, please move this line after the `mod bond;`.

Reply:
Fixed.